### PR TITLE
K8SPXC-595: Change disableHookValidation field in cr

### PR DESCRIPTION
**Pull request by qJkee**
[K8SPXC-595](https://jira.percona.com/browse/K8SPXC-595) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

Change disableHookValidation field in cr

### CODE REVIEW

File: pkg/apis/pxc/v1/pxc_types.go [hunk 1]
```diff
@@ -643,6 +643,11 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *version.ServerV
 
 	cr.setSecurityContext()
 
+	if cr.Spec.EnableCRValidationWebhook == nil {
+		falseVal := false
```
Comment (delgod) [inline pkg/apis/pxc/v1/pxc_types.go:647]: why it is disabled by default? I think it should be enabled

**Review by delgod — CHANGES_REQUESTED** [review]

File: pkg/apis/pxc/v1/pxc_types.go (hunk 1 above)
Comment (qJkee) [reply pkg/apis/pxc/v1/pxc_types.go:647]: it was requested by Sergey
https://jira.percona.com/browse/K8SPXC-312?focusedCommentId=275863&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-275863

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| recreate | passed |
| storage | passed |
| upgrade-haproxy | passed |
| big-data | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| scaling-proxysql | passed |
| limits | passed |
| security-context | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| smart-update | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-721-777b58b7

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| security-context | passed |
| big-data | passed |
| haproxy | passed |
| init-deploy | passed |
| limits | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| upgrade-haproxy | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| storage | passed |
| tls-issue-cert-manager | passed |
| self-healing | passed |
| tls-issue-cert-manager-ref | passed |
| self-healing-advanced | passed |
| validation-hook | passed |
| operator-self-healing | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| scheduled-backup | passed |

File: pkg/apis/pxc/v1/pxc_types.go (hunk 1 above)
Comment (spron-in) [reply pkg/apis/pxc/v1/pxc_types.go:647]: as discussed in this release we are going to disable it by default.

**Review by delgod — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**